docs/overrides/partials/copyright.html

```diff
@@ -0,0 +1,40 @@
+<!--
+  Copyright (c) 2016-2024 Martin Donath <martin.donath@squidfunk.com>
```

Comment:
It's copyright from mkdocs author.

Reply:
@mloiseleur Do I remove it? Or do I ask the author first, to which extent this copyright could be removed? I am not an expert on this matter but, I think overridden templates fall into his copyrighted material.

Reply:
Nope, just a comment for other reviewer :).